package.json

```diff
@@ -26,13 +29,16 @@
     "coverage:ci": "node packages/build/bin/run-nyc report --reporter=text-lcov | coveralls",
     "precoverage": "npm test",
     "coverage": "open coverage/index.html",
-    "lint": "npm run prettier:check && npm run tslint",
-    "lint:fix": "npm run prettier:fix && npm run tslint:fix",
+    "lint": "npm run prettier:check && npm run tslint && npm run lint-md",
+    "lint:fix": "npm run prettier:fix && npm run tslint:fix && npm run lint-md:fix",
+    "lint-md": "remark -f -q .",
+    "lint-md:fix": "npm run prettier-md:fix && remark -S . -o",
```

Comment:
if we fail on warnings, then perhaps we want to use `-q` instead of `-S`?

Reply:
When remark is run in output mode, it will list all warnings (including the ones it fixed). Since `prettier` changes unordered lists from `-` to `*`, it generates an excessive amount of warnings. That said I agree that it'd be helpful to see a list of violations that need to be fixed manually once everything has been run ... A solution that comes to mind would be to run `remark` twice? 

`"lint-md:fix": "npm run prettier-md:fix && remark -S . -o && npm run lint-md"`

Thoughts?

Reply:
ah that makes sense. I can't think of another solution, but the one you proposed sounds good to me. Perhaps others might different perspectives.